src/Microsoft.AspNetCore.SignalR/DefaultHubLifetimeManager.cs

```diff
@@ -104,7 +104,7 @@ public override Task InvokeGroupAsync(string groupName, string methodName, objec
         public override Task InvokeUserAsync(string userId, string methodName, object[] args)
         {
             return InvokeAllWhere(methodName, args, connection => 
-                string.Equals(connection.User.FindFirst(ClaimTypes.NameIdentifier).Value, userId, StringComparison.Ordinal));
+                string.Equals(connection.User.FindFirstValue(ClaimTypes.NameIdentifier), userId, StringComparison.Ordinal));
```

Comment:
How about `connection.User.FindFirst(ClaimTypes.NameIdentifier)?.Value` and not taking the dependency on Microsoft.Extensions.Identity.Core?